jsch/pom.xml

```diff
@@ -73,12 +72,13 @@
         <connection>scm:git:git@github.com:javaee/repackaged.git</connection>
         <developerConnection>scm:git:git@github.com:javaee/repackaged.git</developerConnection>
         <url>https://github.com/javaee/repackaged.git</url>
-    </scm>
+      <tag>HEAD</tag>
```

Comment:
Why is the tag needed?

Reply:
Don't think it's needed. The default value of this is HEAD.